tests/integration/test_start_available_mip_dna.py

```diff
@@ -0,0 +1,319 @@
+from datetime import datetime
+from pathlib import Path
+from subprocess import CompletedProcess
+from time import sleep
+from typing import Literal, cast
+from unittest.mock import ANY, Mock, create_autospec
+
+import pytest
+from click.testing import CliRunner, Result
+from housekeeper.store import database as hk_database
+from housekeeper.store.models import Bundle, Version
+from housekeeper.store.store import Store as HousekeeperStore
+from pytest import TempPathFactory
+from pytest_httpserver import HTTPServer
+
+from cg.apps.tb.api import IDTokenCredentials
+from cg.cli.base import base
+from cg.cli.workflow.microsalt.base import run
+from cg.cli.workflow.mip import base as mip_base
+from cg.constants.constants import CaseActions, Workflow
+from cg.constants.gene_panel import GenePanelMasterList
+from cg.constants.housekeeper_tags import SequencingFileTag
+from cg.constants.process import EXIT_SUCCESS
+from cg.constants.tb import AnalysisType
+from cg.meta.workflow import analysis
+from cg.store import database as cg_database
+from cg.store.models import Case, IlluminaFlowCell, IlluminaSequencingRun, Order, Sample
+from cg.store.store import Store
+from cg.utils import commands
+from tests.store_helpers import StoreHelpers
+
+
+@pytest.fixture(autouse=True)
+def valid_google_token(mocker):
+    mocker.patch.object(
+        IDTokenCredentials,
+        "from_service_account_file",
+        return_value=create_autospec(IDTokenCredentials, token="some token"),
+    )
+
+
+@pytest.fixture(scope="session")
+def httpserver_listen_address():
+    return ("localhost", 8888)
+
+
+@pytest.fixture
+def status_db_uri() -> str:
+    return "sqlite:///file:cg?mode=memory&cache=shared&uri=true"
+
+
+@pytest.fixture
+def housekeeper_db_uri() -> str:
+    return "sqlite:///file:housekeeper?mode=memory&cache=shared&uri=true"
+
+
+@pytest.fixture
+def store(status_db_uri: str) -> Generator[Store, None, None]:
+    cg_database.initialize_database(status_db_uri)
+    cg_database.create_all_tables()
+    store = Store()
+    yield store
+    cg_database.drop_all_tables()
+
+
+@pytest.fixture
+def status_db(store: Store) -> Generator[Store, None, None]:
+    yield store
+
+
+@pytest.fixture
+def housekeeper_db(
+    housekeeper_db_uri: str, tmp_path_factory: TempPathFactory
+) -> Generator[HousekeeperStore, None, None]:
+    hk_database.initialize_database(housekeeper_db_uri)
+    hk_database.create_all_tables()
+    housekeeper_root: Path = tmp_path_factory.mktemp("housekeeper")
+    store = HousekeeperStore(root=housekeeper_root.as_posix())
+    yield store
+    hk_database.drop_all_tables()
+
+
+@pytest.fixture
+def scout_export_panel_stdout() -> bytes:
+    return b"22\t26995242\t27014052\t2397\tCRYBB1\n22\t38452318\t38471708\t9394\tPICK1\n"
+
+
+@pytest.mark.integration
+def test_start_available_mip_dna(
+    helpers: StoreHelpers,
+    housekeeper_db_uri: str,
+    housekeeper_db: HousekeeperStore,
+    httpserver: HTTPServer,
+    mocker: Callable,
```

Comment:
For me, this removes a warning
```suggestion
    mocker: MockerFixture,
```